### PR TITLE
New nimble icons - Play, Pause, Stop, etc.

### LINKED ISSUE #1789: Add Icons to token library (Anchor, Run, Running, Pause, Stop)
Please add and update the following icons to the token library.

Add:

anchor-link_16x16
tags: square, arrow, open, box

Image

run-triangle_16x16
tags: arrow, play, go

Image

pause_16x16
tags: stop, hold 

Image

stop-square_16x16
tags: quit, end

Image

UPDATE:

running-arrow_16x16
tags: play, continue, run

Image

### PR DESCRIPTION
**Pull request by fredvisser**
# Pull Request

## 🤨 Rationale

New icons from visual design. Fixes #1789.

## 👩‍💻 Implementation

Followed [contributing docs](file:///www.example.org)

## 🧪 Testing

Manual and Chromatic reviews.

## ✅ Checklist

- [x] I have updated the project documentation to reflect my changes or determined no changes are needed.

### CODE REVIEW

**Review by rajsite — APPROVED** [review]

File: packages/nimble-tokens/dist/icons/svg/running-arrow_16x16.svg
Comment (jattasNI) [inline packages/nimble-tokens/dist/icons/svg/running-arrow_16x16.svg:1]: The GitHub diff tool for image changes is super cool! I was about to tell you that you uploaded an identical image until I inspected it there.

https://github.com/ni/nimble/assets/10500124/8cbf60c8-ef61-496f-819c-213678e3f102

File: change/@ni-nimble-components-2ca4a1a1-f437-4b1d-be20-f3e0e4756771.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (jattasNI) [inline change/@ni-nimble-components-2ca4a1a1-f437-4b1d-be20-f3e0e4756771.json:1]: Reminder to link to the relevant issue in the PR description. Are you going to track the 2-tone and fixed color requests separately, keep that issue around to do so, or deprioritize it and leave it untracked for now?

File: packages/nimble-tokens/dist/icons/svg/running-arrow_16x16.svg
Comment (fredvisser) [reply packages/nimble-tokens/dist/icons/svg/running-arrow_16x16.svg:1]: Ah! I downloaded them and flipped between windows until I could see the difference, but this is easier.

File: change/@ni-nimble-components-2ca4a1a1-f437-4b1d-be20-f3e0e4756771.json (hunk 1 above)
Comment (fredvisser) [reply change/@ni-nimble-components-2ca4a1a1-f437-4b1d-be20-f3e0e4756771.json:1]: 👍 and tracked as #1802

**Pull request merged**